src/Domain/DomainHelpers.cpp

```diff
@@ -503,18 +503,52 @@ wedge_coordinate_maps(const double inner_radius, const double outer_radius,
                       const double x_coord_of_shell_center,
                       const bool use_half_wedges, const double aspect_ratio,
                       const bool use_logarithmic_map,
-                      const ShellWedges which_wedges) noexcept {
+                      const ShellWedges which_wedges,
+                      const size_t number_of_layers) noexcept {
+  ASSERT(use_half_wedges ? (which_wedges == ShellWedges::All) : true,
```

Comment:
`not use_half_wedges or which_wedges == ShellWedges::All`  and similar for the others.

Reply:
Also `x == false` -> `not x` later.

Reply:
Let me know if I can squash :smile: